integration_tests/__tests__/__snapshots__/custom-reporters-test.js.snap

```diff
@@ -0,0 +1,80 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Custom Reporters Integration IncompleteReporter for flexibility 1`] = `
+"onRunComplete is called
+Passed Tests: 1
+Failed Tests: 0
+Total Tests: 1
+"
+`;
+
+exports[`Custom Reporters Integration TestReporter with all tests failing 1`] = `
+Object {
+  "onRunComplete": Object {
+    "called": true,
+    "config": "object",
+  },
+  "onRunStart": Object {
+    "called": true,
+    "config": "object",
+    "options": "object",
+  },
+  "onTestResult": Object {
+    "called": true,
+    "times": 1,
+  },
+  "onTestStart": Object {
+    "called": true,
+    "config": false,
+    "path": false,
+  },
+  "options": Object {
+    "christop": "pojer",
+    "dmitrii": "abramov",
+    "hello": "world",
+  },
+}
+`;
+
+exports[`Custom Reporters Integration TestReporter with all tests passing 1`] = `
+Object {
+  "onRunComplete": Object {
+    "called": true,
+    "config": "object",
+  },
+  "onRunStart": Object {
+    "called": true,
+    "config": "object",
+    "options": "object",
+  },
+  "onTestResult": Object {
+    "called": true,
+    "times": 1,
+  },
+  "onTestStart": Object {
+    "called": true,
+    "config": false,
+    "path": false,
+  },
+  "options": Object {
+    "christop": "pojer",
+    "dmitrii": "abramov",
+    "hello": "world",
+  },
+}
+`;
+
+exports[`Custom Reporters Integration invalid format for adding reporters 1`] = `
+"●  Reporter Validation Error:
+
+Unexpected value for Path at index 0 of reporter at index 0
```

Comment:
This message is so not nice. `index 0 of reporter at index 0`. Can we maybe print actual reporters value along with it?